### PR TITLE
add help text (and links) to a11y audits

### PR DESCRIPTION
**Pull request by kaycebasques**
WDYT @robdodson PTAL LGTY?

### CODE REVIEW

**Comment by robdodson** [comment]
Seems OK to me. I'm aware from the computer for a bit but I'll check it in my smoke test when I get back. Thanks for doing this!

**Comment by robdodson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: lighthouse-core/audits/accessibility/image-alt.js [hunk 1]
```diff
@@ -33,6 +33,9 @@ class ImageAlt extends AxeAudit {
       category: 'Accessibility',
       name: 'image-alt',
       description: 'Every image element has an alt attribute',
+      helpText: 'Screen readers can\'t read images to their users. The `alt` ' +
```
Comment (robdodson) [inline lighthouse-core/audits/accessibility/image-alt.js]: Suggest changing to something like "Screen reader users rely on alt text to provide descriptions for images. It's also used as fallback content if the image fails to load"

File: lighthouse-core/audits/accessibility/color-contrast.js [hunk 2]
```diff
@@ -34,6 +34,9 @@ class ColorContrast extends AxeAudit {
       category: 'Accessibility',
       name: 'color-contrast',
       description: 'Background and foreground colors have a sufficient contrast ratio',
+      helpText: 'Some users with low vision have a difficult time sensing contrast. ' +
```
Comment (robdodson) [inline lighthouse-core/audits/accessibility/color-contrast.js]: Suggest rewording to "Low contrast text is difficult or impossible for many users to read."

File: lighthouse-core/audits/accessibility/aria-allowed-attr.js [hunk 3]
```diff
@@ -33,6 +33,8 @@ class ARIAAllowedAttr extends AxeAudit {
       category: 'Accessibility',
       name: 'aria-allowed-attr',
       description: 'Element aria-* attributes are allowed for this role',
+      helpText: 'Invalid role-attribute combinations can disable the accessibility ' +
```
Comment (robdodson) [inline lighthouse-core/audits/accessibility/aria-allowed-attr.js]: Suggest rewording to something like "Each ARIA `role` defines a specific subset of `aria-*` attributes that it supports. Mismatching these will invalidate the aria-* attributes."

File: lighthouse-core/audits/accessibility/aria-required-attr.js [hunk 4]
```diff
@@ -33,6 +33,8 @@ class ARIARequiredAttr extends AxeAudit {
       category: 'Accessibility',
       name: 'aria-required-attr',
       description: 'Elements with ARIA roles have the required aria-* attributes',
+      helpText: 'Some ARIA roles have required attributes that describe the state ' +
+          'of the role to screen readers. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/required-aria-attributes).',
```
Comment (robdodson) [inline lighthouse-core/audits/accessibility/aria-required-attr.js]: Might change this to "state of the element".

File: lighthouse-core/audits/accessibility/aria-valid-attr-value.js [hunk 5]
```diff
@@ -33,6 +33,8 @@ class ARIAValidAttr extends AxeAudit {
       category: 'Accessibility',
       name: 'aria-valid-attr-value',
       description: 'Element aria-* attributes have valid values',
+      helpText: 'Screen readers may not be able to read elements with invalid ARIA ' +
```
Comment (robdodson) [inline lighthouse-core/audits/accessibility/aria-valid-attr-value.js]: Suggest rewording to something like "Assistive technology, such as a screen reader, will not be able to interpret ARIA attributes with invalid values"

File: lighthouse-core/audits/accessibility/aria-valid-attr.js [hunk 6]
```diff
@@ -33,6 +33,8 @@ class ARIAValidAttr extends AxeAudit {
       category: 'Accessibility',
       name: 'aria-valid-attr',
       description: 'Element aria-* attributes are valid and not misspelled or non-existent.',
+      helpText: 'Screen readers may not be able to read elements with invalid ARIA ' +
```
Comment (robdodson) [inline lighthouse-core/audits/accessibility/aria-valid-attr.js]: Suggest rewording to something like "Assistive technology, such as a screen reader, will not be able to interpret ARIA attributes with invalid attribute names"

File: lighthouse-core/audits/accessibility/label.js [hunk 7]
```diff
@@ -33,6 +33,9 @@ class Label extends AxeAudit {
       category: 'Accessibility',
       name: 'label',
       description: 'Every form element has a label',
+      helpText: 'Although the purpose of each form element may be obvious ' +
```
Comment (robdodson) [inline lighthouse-core/audits/accessibility/label.js]: Suggest rewording to "Labels ensure that form controls are announced properly by assistive technology, such as a screen reader"

File: lighthouse-core/audits/accessibility/image-alt.js (hunk 1 above)
Comment (kaycebasques) [inline lighthouse-core/audits/accessibility/image-alt.js]: Done

File: lighthouse-core/audits/accessibility/color-contrast.js (hunk 2 above)
Comment (kaycebasques) [inline lighthouse-core/audits/accessibility/color-contrast.js]: Done

File: lighthouse-core/audits/accessibility/aria-allowed-attr.js (hunk 3 above)
Comment (kaycebasques) [inline lighthouse-core/audits/accessibility/aria-allowed-attr.js]: Done

File: lighthouse-core/audits/accessibility/aria-required-attr.js (hunk 4 above)
Comment (kaycebasques) [inline lighthouse-core/audits/accessibility/aria-required-attr.js]: Done

File: lighthouse-core/audits/accessibility/aria-valid-attr.js (hunk 6 above)
Comment (kaycebasques) [inline lighthouse-core/audits/accessibility/aria-valid-attr.js]: Done

File: lighthouse-core/audits/accessibility/aria-valid-attr-value.js (hunk 5 above)
Comment (kaycebasques) [inline lighthouse-core/audits/accessibility/aria-valid-attr-value.js]: Done

File: lighthouse-core/audits/accessibility/label.js (hunk 7 above)
Comment (kaycebasques) [inline lighthouse-core/audits/accessibility/label.js]: Done

**Comment by kaycebasques** [comment]
Hallo mr @robdodson "updates were made"
